docs.html

```diff
@@ -261,35 +628,103 @@ <h6>Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, adipisci vel
             </div>
         </div> 
     </div>
+<pre>
+<code class="html">
+
+    <div class="m1 card flex-r horizontal-card-container">
+        <div class="badge-container">
+            <span class="text-badge">NEW</span>
+            <img src="./images/croton.jpg" alt="croton" class="card-img">
+        </div>
+        <div class="card-text-container">
+            <h5 class="card-header">Croton Plant</h5>
+            <div class="card-info">Stock available</div>
+            <p class="card-price">Rs. 499</p>
+            <h6>Neque porro quisquam est qui dolorem ipsum quia 
+                dolor sit amet, adipisci velit...</h6>
+            <div class="card-btns align-center">
+                <button class="btn icon-inside-btn">
+                    <i class="p1-right fas fa-shopping-cart"></i>Add to cart</button>
+                <button class="m1 btn primary-outline-btn">Move to Wishlist</button>           
+            </div>
+        </div> 
     </div>
 
+</code>
+</pre>
+<hr>
+</section>
+
 
     <!-- Image Components -->
-    <h4 class="p2-left">Cards</h4>
-    <div class="container">
-        <h6>Responsive Image</h6>
-        <img class="responsive-img" src="./images/res.webp" alt="Responsive img">
-        <h6>Round Image</h6>
-        <img class="round-img" src="./images/succulent.jpg" alt="Succulent">
-    </div>
+    <section id="Image">
+    <h2 class="component-heading">Images</h2>
+    <p class="component-desc">Images can be responsive to fit the parent's width, and also can be customised to be round shaped. </p>
+    <h5 class="component-subheading">Responsive Image</h5>
+    <img class="responsive-img" src="./images/res.webp" alt="Responsive img">
+<pre>
+<code class="html">
+
+    <img class="responsive-img" src="./images/res.webp" alt="Responsive img">
+
+</code>
+</pre>
+<hr>
+
+    <h5 class="component-subheading">Round Image</h5>
+    <img class="round-img" src="./images/succulent.jpg" alt="Succulent">
+<pre>
+<code class="html">
+
+    <img class="round-img" src="./images/succulent.jpg" alt="Succulent">
+
+</code>
+</pre>
+<hr>
+    </section>
+
 
     <!-- Input Component -->
-    <h4 class="p2-left">Input Component</h4>
-    <div class="container">
-        <h6>Textbox</h6>
+    <section id="Input">
+    <h2 class="component-heading">Input Component</h2>
+    <p class="component-desc">Examples of Textbox, Textbox with error styling, and form is given below.</p>
+    <h5 class="component-subheading">Textbox</h5>
         <div class="m1 input-container">
             <label class="input-label" for="input-text">Username</label>
```

Comment:
for attribute in label and id in the input must be same. So that when you click on the label text, input field auto focuses.